src/components/input-panel/index.tsx

```diff
@@ -36,7 +36,7 @@ class InputPanel extends React.Component<Props> {
   public componentDidUpdate() {
     if (this.props.mode === Mode.VegaLite) {
       if (this.props.compiledVegaPaneSize === 25) {
-        this.props.setCompiledVegaPaneSize(LAYOUT.MinPaneSize + 25);
+        this.props.setCompiledVegaPaneSize((window.innerHeight - LAYOUT.HeaderHeight) * 0.3);
```

Comment:
LAYOUT.MinPaneSize is now unused.

Reply:
`LAYOUT.MinPaneSize` is used some places so can't be removed from `consts.ts`

Reply:
Shouldn't you use it then to determine the min size?

Reply:
Initially, there will just header which is 25px in height. Let's say window.innerHeight is 160px(including the top header). And there is the top header which is 60px in heighr
So when clicked for the first time, it will expand to (160-60)*0.3 = 30px height. 

Do you want me to add a condition which will reset it to 25px if the above expression evaluates to less than 25px?